Expensify/App
# Make the current password field also submittable via Enter

### Details


A very small UI change which makes the `Current password` field actionable via `Enter`.

On the form to change password, this change would enable user to submit the form using `Enter` key also when the `Current password` field would be focused. This is for scenario, when user enters both, current and new password, but realizes they entered the current password wrong, then they can submit the form while focusing the current password too.

If the New password is not filled in, validation does not pass.

### Fixed Issues

$ https://github.com/Expensify/App/issues/6823

### Tests

- Go to the `Change password page`
- Type in your Current password
- Type in your New Password
- Focus back to the Current password field and hit enter. 
- Make sure the form has been submitted

### QA Steps

- Go to the `Change password page`
- Type in your Current password
- Type in your New Password
- Focus back to the Current password field and hit enter. 
- Make sure the form has been submitted

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


![image](https://user-images.githubusercontent.com/36083550/147786172-3fbcc575-0ba2-4b14-a80f-3e727f7928a1.png)


#### iOS


#### Android
